.gitignore

```diff
@@ -29,4 +29,4 @@ hie.yaml
 
 ### Nix ###
 result*
-.stack-to-nix.cache
+.stack-to-nix.cache
```

Comment:
Your editor appears to be ignoring `.editorconfig`.
But at least it's not saving files with windows line endings.

Reply:
Plugin installed.